runtime/gc_glue_java/ObjectModelDelegate.hpp

```diff
@@ -461,6 +461,14 @@ class GC_ObjectModelDelegate
 	void calculateObjectDetailsForCopy(MM_EnvironmentBase *env, MM_ForwardedHeader *forwardedHeader, uintptr_t *objectCopySizeInBytes, uintptr_t *objectReserveSizeInBytes, uintptr_t *hotFieldAlignmentDescriptor);
 #endif /* defined(OMR_GC_MODRON_SCAVENGER) */
 
+	/**
+	 * Initialize an object of minimal object size at given addr. Specifically, java/lang/object is used.
+	 *
```

Comment:
You can add a comment that the assumption is that the object is not visible (referred) by anyone, so the minimum initialization is done.